test/Atmos/EDMF/closures/mixing_length.jl

```diff
@@ -83,7 +83,9 @@ function mixing_length(
     # Dissipation term
     b = FT(0)
     a_up = vuntuple(i -> up[i].ρa * ρinv, N_up)
-    w_up = vuntuple(i -> up[i].ρaw / up[i].ρa, N_up)
+    w_up = vuntuple(N_up) do i
+        up[i].ρa < eps(FT) ? FT(0) : up[i].ρaw / up[i].ρa
```

Comment:
I might suggest defining `effectively_zero_area_fraction(up_ρa::FT) where {FT} = up_ρa < eps(FT)`, so that we can control the tolerance in one place. I think all of the tolerances should be in sync.